src/main/java/gregtech/api/capability/impl/MultiblockRecipeLogic.java

```diff
@@ -54,12 +60,29 @@ protected IMultipleTankHandler getOutputTank() {
     protected boolean setupAndConsumeRecipeInputs(Recipe recipe) {
         RecipeMapMultiblockController controller = (RecipeMapMultiblockController) metaTileEntity;
         if (controller.checkRecipe(recipe, false) &&
-            super.setupAndConsumeRecipeInputs(recipe)) {
+            multiBlockSetupAndConsumeRecipeInputs(recipe)) {
             controller.checkRecipe(recipe, true);
             return true;
         } else return false;
     }
 
+    //Logic Mostly copied from AbstractRecipeLogic, but with some additional checking for Multiblock output spacing to prevent voiding
+    protected boolean multiBlockSetupAndConsumeRecipeInputs(Recipe recipe) {
+        int[] resultOverclock = calculateOverclock(recipe.getEUt(), getMaxVoltage(), recipe.getDuration());
+        int totalEUt = resultOverclock[0] * resultOverclock[1];
+        IItemHandlerModifiable importInventory = getInputInventory();
+        IItemHandlerModifiable exportInventory = getOutputInventory();
+        IMultipleTankHandler importFluids = getInputTank();
+        IMultipleTankHandler exportFluids = getOutputTank();
+        List<ItemStack> itemOutputs = recipe.getAllItemOutputs(exportInventory.getSlots());
+        return (totalEUt >= 0 ? getEnergyStored() >= (totalEUt > getEnergyCapacity() / 2 ? resultOverclock[0] : totalEUt) :
+            (getEnergyStored() - resultOverclock[0] <= getEnergyCapacity())) &&
+            MetaTileEntity.addItemsToItemHandler(exportInventory, true, itemOutputs) &&
```

Comment:
I took another look at this and this function does exactly what you are doing with new one. But this one does not work correctly when more then one item is given and it takes up last available slot. As it simulates input for every item one by one without considering others.
I think we should fix this function not creating new one that does same thing. But we sure can use knowledge obtained when designing new merge function.

Reply:
This has been done. The logic in addItemsToItemHandler has been adjusted to call the new method logic, and the simulate case from the original addItemsToItemHandler method is also handled.